### PR TITLE
Get screenshots working -- only use one browser in layout tests

### LINKED ISSUE #4558: Fonts in screenshots generated by functional tests are broken
The functional test run produces screenshot artifacts in `tests/pageslayout/screenshots` that can be used for the documentation. However, the fonts in those screenshots are currently broken (see below).

A fix currently exists in this remote branch:
https://github.com/rmol/securedrop/commit/6f493c745a8e0532c7c112f2a5949627875ec0fd

source-login

### PR DESCRIPTION
**Pull request by rmol**
## Status

Ready for review

## Description of Changes

Somehow having two browsers open in the test container resulted in corrupted screenshots, where all text was tofu (undefined unicode glyphs).

Fixes #4558.

## Testing

Run some page layout tests with a few locales and check the screenshots:

- `export PAGE_LAYOUT_LOCALES=ar,el,en_US,fr_FR`  # (Arabic, Greek, English, French)
- `make -C securedrop test TESTFILES=tests/pageslayout/test_journalist.py`
- open the PNG images under `securedrop/pageslayout/screenshots/*/*.png` and confirm that all text is rendered properly. Note that at the moment we have some untranslated strings on `develop`, so it's not necessarily a problem to find unexpected English text in some of the screenshots.

## Deployment

This only involves the functional tests. There are no deployment concerns.

### If you made changes to the server application code:

- [x] Linting (`make lint`) and tests (`make -C securedrop test`) pass in the development container

### If you made non-trivial code changes:

- [x] I have written a test plan and validated it for this PR

### CODE REVIEW

**Comment by eloquence** [comment]
These tests failed for me, not sure if known flakes:

```
tests/pageslayout/test_journalist.py::TestJournalistLayout::test_delete_one[el] FAILED            [ 62%]
tests/pageslayout/test_journalist.py::TestJournalistLayout::test_delete_one[fr_FR] FAILED         [ 63%]
```

Now reviewing screenshots.

**Comment by eloquence** [comment]
With the above test failure caveat, all generated screenshots rendered correctly for me in the languages in the test plan.

This shouldn't be a blocker for merging this PR, but noting that for use in the docs, the following currently require manual editing or re-taking:

Bad cropping:
- `journalist-composes_reply.png` cut off just above the submit button
- `source-flagged.png` cut off just above the submit button
- `source-lookup.png` has bits of the "Read Replies" text cut off
- `source-checks_for_reply.png` does not show the reply
- `source-flagged.png` has the "Submit" button cut off

Too much whitespace below the footer:
- basically all screenshots of short pages

Bad content:
- `source-index.png` has JavaScript warning, but JS should be disabled for this screenshot in the docs
- `source-logout_flashed_message.png` has JavaScript warning, and does not in fact show a flashed message (guessing because it's FF not Tor in this case? it's styled to render in Tor only)

**Comment by eloquence** [comment]
Those two exact tests failed for me again on a second run, detailed test output here:
https://gist.github.com/eloquence/5b3bcfc449235199630e1eb0033f1473
